src/index.html

```diff
@@ -11,8 +11,64 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Product cards</h1>
+
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__photo"
+        src="./images/imac.jpeg"
```

Comment:
Verify that the image path is correct and that the image `imac.jpeg` exists in the `images` directory. This is important to ensure the image displays correctly on the page.